packages/react-native/ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -467,7 +467,7 @@ inline Float toRadians(
   if (!value.hasType<std::string>() && defaultValue.has_value()) {
     return *defaultValue;
   }
-  auto stringValue = (std::string)value;
+  auto stringValue = std::string_view{(std::string)value};
   char* suffixStart;
   double num = strtod(
       stringValue.c_str(), &suffixStart); // can't use std::stod, probably
```

Comment:
We also need to replace `.c_str()` with `.data()`
```suggestion
      stringValue.data(), &suffixStart); // can't use std::stod, probably
```